### ISSUE TITLE
Unable to resolve peer dependency `reflect-metadata@"^0.1.13"`

### ISSUE THREAD

**Issue by BayBreezy** [dmitriy-nz/nestjs-form-data#58]
Hello,

When I try to install the package with the latest version of Nest(v10.3.3), I get this error message

```bash
npm ERR! While resolving: wealth-api-nestjs@0.0.1
npm ERR! Found: reflect-metadata@0.2.1
npm ERR! node_modules/reflect-metadata
npm ERR!   reflect-metadata@"^0.2.1" from the root project
npm ERR!   peer reflect-metadata@"^0.1.12 || ^0.2.0" from @nestjs/common@10.3.3
npm ERR!   node_modules/@nestjs/common
npm ERR!     @nestjs/common@"^10.3.3" from the root project
npm ERR!     peer @nestjs/common@"^7.0.0 || ^8.0.0 || ^9.0.0 || ^10.0.0" from nestjs-form-data@1.9.5
npm ERR!     node_modules/nestjs-form-data
npm ERR!       nestjs-form-data@"*" from the root project
npm ERR!     5 more (@nestjs/core, @nestjs/microservices, ...)
npm ERR!   3 more (@nestjs/core, @nestjs/microservices, @nestjs/websockets)
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer reflect-metadata@"^0.1.13" from nestjs-form-data@1.9.5
npm ERR! node_modules/nestjs-form-data
npm ERR!   nestjs-form-data@"*" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

I can install it with force for now

**Comment by dmitriy-nz** [comment]
Hey, thanks for reporting!   
This has been resolved in version [1.9.6](https://github.com/dmitriy-nz/nestjs-form-data/releases/tag/v1.9.6)

**Comment by BayBreezy** [comment]
Thank you 😄
